TeaLeaf/jni/libpng/pngget.c

```diff
@@ -862,9 +862,10 @@ png_get_sCAL_fixed(png_structp png_ptr, png_const_infop info_ptr,
    {
       *unit = info_ptr->scal_unit;
       /*TODO: make this work without FP support */
-      *width = png_fixed(png_ptr, atof(info_ptr->scal_s_width), "sCAL width");
-      *height = png_fixed(png_ptr, atof(info_ptr->scal_s_height),
-         "sCAL height");
+      *width = png_fixed(png_ptr, strtod(info_ptr->scal_s_width, NULL),
```

Comment:
yes but atof wont be present in devices with less than android-21 if native target is >=21